iambic/config/wizard.py

```diff
@@ -1875,7 +1876,9 @@ def configuration_wizard_change_detection_setup(self, aws_org: AWSOrganization):
 
         role_name = IAMBIC_SPOKE_ROLE_NAME
         hub_account_id = self.hub_account_id
-        sqs_arn = f"arn:aws:sqs:{self.aws_default_region}:{hub_account_id}:IAMbicChangeDetectionQueue{IAMBIC_CHANGE_DETECTION_SUFFIX}"
+        # cloudtrail is not cross-region, so we need to use us-east-1
```

Comment:
We'll need to check if SSO messages are duplicated or if we're going to need a dedicated queue for SSO.

It may just require a ticket and not need to worry about fixing in this PR.